Dockerfile

```diff
@@ -2,9 +2,9 @@
 # We don't declare them here — take a look at our docs.
 # https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md
 
-FROM nginx:1.19-alpine
+FROM nginx:1-alpine
```

Comment:
By fixing major nginx version you ensure to get last nginx fixes available on each build. Since this is the "major" version fixed, this is unlikely you get regression from this. It's essentially increase security response and lighten the work of maintainers.

Imho this rule of fixing tag is coming from docker early age and the idea to have a determinist and reproducible build. In reality, chances that you get the same image if you build both with 6 months interval are rare. Every package would have to be version fixed for this to happen: apk, npm, etc.

I just wanted to elaborate my point. But point taken to split nginx version changed in another PR/commit.

Reply:
A quick note on versioning, Dependabot automatically updates the version to latest available version. Atm, nginx@1.21.6-alpine is current. There was a recent PR that requires nginx@1.19, so it's still better to include the patch version.